secrets/observability-cloudwatchlogs-config.yaml.sample

```diff
@@ -20,3 +20,12 @@ k8s_credentials_secret_name: #fill here
 # Accepted values: ["openshift-logging"]
 # Default value: "openshift-logging"
 k8s_credentials_secret_namespace: #fill here
+
+aws_iam_credentials_enterprise:
+  # [required] org_id
```

Comment:
Can we document what `org_id` is? as done with the other attributes?

Reply:
:+1: